pkg/sql/delegate/show_completions_test.go

```diff
@@ -30,8 +31,7 @@ func TestCompletions(t *testing.T) {
 			expectedCompletions: []string{"CREATE", "CREATEDB", "CREATELOGIN", "CREATEROLE"},
 		},
 		{
-			stmt:                "creat ",
-			expectedCompletions: []string{},
```

Comment:
nit: seems useful to be explicit that the expected completions are an empty slice (and same for below tests)

Reply:
The function returns `nil`, not empty, which made the assertion confusing. I will add explicit `nil` though.